CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+# 108.2.0
+
+This minor release deletes bazel support, improves documentation for Cards and TextControls, and
+adds the large content view support for MDCItemBar.
```

Comment:
Deleting bazel support counts as a breaking change given that nobody will be able to build MDC with bazel anymore.

Reply:
That makes sense. I bumped the version to 109.0.0 because of this breaking change.